server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/UnsetDefaultAction.java

```diff
@@ -19,28 +19,29 @@
  */
 package org.sonar.server.qualitygate.ws;
 
+import org.sonar.api.server.ws.Change;
 import org.sonar.api.server.ws.Request;
 import org.sonar.api.server.ws.Response;
 import org.sonar.api.server.ws.WebService;
-import org.sonar.server.qualitygate.QualityGates;
+import org.sonar.server.ws.RemovedWebServiceHandler;
 import org.sonarqube.ws.client.qualitygate.QualityGatesWsParameters;
 
 public class UnsetDefaultAction implements QualityGatesWsAction {
 
-  private final QualityGates qualityGates;
-
-  public UnsetDefaultAction(QualityGates qualityGates) {
-    this.qualityGates = qualityGates;
-  }
-
   @Override
   public void define(WebService.NewController controller) {
     WebService.NewAction action = controller.createAction("unset_default")
       .setDescription("Unset a quality gate as the default quality gate.<br>" +
```

Comment:
Please update the description explaining that this action is now useless.
Something like "This web service has no effect since 7.0. It's no more possible to have not default quality gate".
Because otherwise, in the UI, you only see this WS as deprecated, but you still see in the description that it's doing something, which is no more the case